### PR TITLE
[daint-gpu] mpip

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sources = [{
+    'filename': 'mpiP-57fc864.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/LLNL',
+        'repo_name': 'mpiP',
+        'commit': '57fc864',
+    }
+}]
+
+configopts = ' --with-libunwind=$EBROOTLIBUNWIND '
```
Comment (kraushm) [inline easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb]: I would add: `start_dir = 'mpiP'`

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+prebuildopts = 'cd mpiP ;'
```
Comment (kraushm) [inline easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb]: Then I would remove this line.

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+buildopts = ' CC=cc F77=ftn '
+preinstallopts = 'cd mpiP ;'
```
Comment (kraushm) [inline easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb]: Then I would remove this line.

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayGNU-18.08.eb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    'filename': 'mpiP-57fc864.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/LLNL',
+        'repo_name': 'mpiP',
+        'commit': '57fc864',
+    }
+}]
+
+configopts = ' --with-libunwind=$EBROOTLIBUNWIND '
+prebuildopts = 'cd mpiP ;'
```
Comment (kraushm) [inline easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayGNU-18.08.eb]: Same as for easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayIntel-18.08.eb [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    'filename': 'mpiP-57fc864.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/LLNL',
+        'repo_name': 'mpiP',
+        'commit': '57fc864',
+    }
+}]
+
+configopts = ' --with-libunwind=$EBROOTLIBUNWIND '
+prebuildopts = 'cd mpiP ;'
```
Comment (kraushm) [inline easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayIntel-18.08.eb]: Same as for easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayPGI-18.08.eb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+sources = [{
+    'filename': 'mpiP-57fc864.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/LLNL',
+        'repo_name': 'mpiP',
+        'commit': '57fc864',
+    }
+}]
+
+configopts = ' --with-libunwind=$EBROOTLIBUNWIND '
```
Comment (kraushm) [inline easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayPGI-18.08.eb]: Same as for easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb

File: easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb (hunk 1 above)
Comment (jgphpc) [reply easybuild/easyconfigs/m/mpiP/mpiP-57fc864-CrayCCE-18.08.eb]: 👍 🆗

**Pull request merged**